test/ffmpeg-vaapi/transcode/transcoder.py

```diff
@@ -10,35 +10,132 @@
 
 @slash.requires(have_ffmpeg)
 @slash.requires(have_ffmpeg_vaapi_accel)
+@platform_tags(ALL_PLATFORMS)
 class TranscoderTest(slash.Test):
+  requirements = dict(
+    decode = {
+      "avc" : dict(
+        sw = (ALL_PLATFORMS, (True, True), None),
+        hw = (AVC_DECODE_PLATFORMS, (True, True), None),
```

Comment:
For ffmpeg-vaapi HW encoders, please add a comment that these are builtin and there is no way to query whether or not they are available.

Reply:
added the comment